<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>980</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Similarity Explorer</string>
  </property>
  <widget class="QTableView" name="uiSimilarityTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>1226</width>
     <height>906</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>1240</x>
     <y>0</y>
     <width>676</width>
     <height>976</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>674</width>
      <height>974</height>
     </rect>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>10</y>
       <width>361</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="uiToolbarLayout"/>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>75</y>
       <width>666</width>
       <height>896</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="uiPlotLayout"/>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>371</x>
       <y>11</y>
       <width>301</width>
       <height>61</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1" colspan="2">
       <widget class="QLabel" name="uiSilhouetteLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="uiNumClustersSpin">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="minimum">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="uiTLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Number of clusters</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QComboBox" name="uiFPTypeCombo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>24</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Fingerprint type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>6</y>
     <width>201</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Similarity metric</string>
   </property>
  </widget>
  <widget class="QComboBox" name="uiSimilMetricCombo">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>25</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="uiHeatmapButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>25</y>
     <width>196</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Heatmap</string>
   </property>
  </widget>
  <widget class="QPushButton" name="uiShowSimilMapButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>665</x>
     <y>25</y>
     <width>186</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Similarity Map</string>
   </property>
  </widget>
  <zorder>scrollArea</zorder>
  <zorder>uiSimilarityTable</zorder>
  <zorder>uiFPTypeCombo</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>uiSimilMetricCombo</zorder>
  <zorder>uiHeatmapButton</zorder>
  <zorder>uiShowSimilMapButton</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>uiFPTypeCombo</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>set_fp_type()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>-8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uiSimilMetricCombo</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>set_simil_metric()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>-2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uiHeatmapButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>show_heatmap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>556</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>-16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uiNumClustersSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>get_HCL()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1854</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>1821</x>
     <y>-16</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>set_fp_type()</slot>
  <slot>set_simil_metric()</slot>
  <slot>show_heatmap()</slot>
  <slot>show_simil_map()</slot>
  <slot>get_HCL()</slot>
 </slots>
</ui>
